src/app/api/snapshot/route.ts

```diff
@@ -75,6 +84,10 @@ export async function GET() {
         });
       }
 
+      console.log(
+        `Snapshot cron executed. Updated: ${proposals.length} proposals`
+      );
```

Comment:
@Flip-Liquid wondering if this is the right way to log this?

Reply:
For now, this will work. but likely, we'll want to use a common cron logger or some such logger with common attributes.